index.html

```diff
@@ -721,6 +721,62 @@ <h2>
             </p>
           </div>
         </section>
+        <section>
+          <h2><dfn>CurrentTabPreferenceEnum</dfn></h2>
+          <p>
+            Recall that when offering the user a choice of [=display surfaces=]
+            to capture, the user agent MAY offer the [=browser=] [=display surface=]
+            in which the application is running. An application might prefer that
+            this particular [=display surface=] not be offered to the user.
+            The CurrentTabPreferenceEnum enumeration describes the different
+            preferences an application might have.
+          </p>
+          <div class="note">
+            <p>
+              For clarity's sake, "tab" is used rather than "browser display surface".
+            </p>
+          </div>
+          <pre class="idl">
+            enum CurrentTabPreferenceEnum {
+              "none",
```

Comment:
`currentTabPreference: "none"` might be confused with `currentTabPreference: "exclude"`. `currentTabPreference: "no-preference"` seems clearer.

I can't decide whether this third option is cleaner or redundant, since this is input only. I couldn't find any design guidance on it.

One thing to be aware of is that extending enums later is tricky (because old browsers will throw on new values). So it's easier to add now and remove later than the opposite.
```suggestion
              "no-preference",
```